utils/node_local_test.py

```diff
@@ -1740,7 +1729,7 @@ def readdir_test(self, count, test_all=False):
             return
         start = time.time()
         for idx in range(count):
-            fd = open(os.path.join(wide_dir, str(idx)), 'w')
+            fd = open(join(wide_dir, str(idx)), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations